src/unix-manager.c

```diff
@@ -1011,6 +1011,19 @@ void UnixManagerThreadSpawn(int mode)
     return;
 }
 
+// TODO can't think of a good name
+void UnixManagerThreadSpawnNonRunmode(void)
```

Comment:
Live mode suggests live traffic capture. We also spawn for offline pcap reading.